### PR TITLE
feat(docs): FRMW-192 - Migrate rxSelect docs

### PR DESCRIPTION
**Pull request by bry4n**
JIRA: https://jira.rax.io/browse/FRMW-192

* [x] Dev LGTM
* [x] Dev LGTM

### CODE REVIEW

**Comment by ssalinas24** [comment]
LGTM

File: src/rxSelect/rxSelect.js [hunk 1]
```diff
@@ -4,7 +4,8 @@
  * @description
  * # rxSelect component
  *
- * [TBD]
+ * The rxSelect component is an **attribute directive** that wraps native select
```
Comment (CITguy) [inline src/rxSelect/rxSelect.js]: The component itself is not a directive. I'd go with something similar with what is in `rxCheckbox`:

```
A component that wraps a native `<select>` element in markup required for styling purposes.
```

Comment (bry4n) [inline src/rxSelect/rxSelect.js]: updated

File: src/rxSelect/rxSelect.js [hunk 2]
```diff
@@ -4,7 +4,8 @@
  * @description
  * # rxSelect component
  *
- * [TBD]
+ * A component that wraps a native `<select>` element in markup required for
+ * styling purposes.
```
Comment (CITguy) [inline src/rxSelect/rxSelect.js]: Please update README with same description.

Comment (bry4n) [inline src/rxSelect/rxSelect.js]: oops forget about that. need :coffee:

**Comment by 100stacks** [comment]
LGTM

**Pull request merged**